tests/publiccloud/download_repos.pm

```diff
@@ -55,12 +56,21 @@ sub run {
         assert_script_run('touch /tmp/repos.list.txt');
 
         my $ret = 0;
+        my $reject = "'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*'";
+        my $regex = "'s390x\\/|ppc64le\\/|kernel*debuginfo*.rpm|src\\/'";
         for my $maintrepo (@repos) {
-            next if $maintrepo !~ m/^http/;
```

Comment:
what about this check ? looks like we missing it now , was that intentional ?

Reply:
Well it was intentional, because:
1) I don't think that there will be problems and if so, then:
2) We were just blindly ignoring it until now.
3) If there will be problem with it, we will notice it and will be able to solve it.